CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+## [2.2.0] - 2021-08-05
```

Comment:
realised that I have to update the package version in the PR too for this repo, so I'll change this to a minor release

Reply:
Done!